src/vinyldns/client.py

```diff
@@ -645,13 +645,14 @@ def approve_batch_change(self, batch_change_id, review=None):
 
         return BatchChange.from_dict(data)
 
-    def reject_batch_change(self, batch_change_id, review_comment=None):
+    def reject_batch_change(self, batch_change_id, review=None):
```

Comment:
Can we rename `review` to be clearer here? Maybe something like `rejection_body` or something.

Reply:
should it be `review_comments`?